server/game/costs.js

```diff
@@ -527,10 +527,10 @@ const Costs = {
     playLimited: function() {
         return {
             canPay: function(context) {
-                return !context.source.isLimited() || context.player.limitedPlayed < context.player.maxLimited;
+                return !context.source.hasPrintedKeyword('limited') || context.player.limitedPlayed < context.player.maxLimited;
             },
             pay: function(context) {
-                if(context.source.isLimited()) {
+                if(context.source.hasPrintedKeyword('limited')) {
```

Comment:
This and the check above should check whether it `isLimited()` OR `hasPrintedKeyword('limited')` in case there's ever an effect that, for instance, adds the Limited keyword to cards in hand (similar to the way certain cards add Ambush). It's unlikely to happen, but would rather not be caught flat footed.